src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -320,7 +324,8 @@ private function getModuleCache($file, $checkFreshness = true)
     private function registerModuleCache($file, $data)
     {
         try {
-            $cache = (new ConfigCacheFactory(true))->cache($this->cache_dir.$file, function () {});
```

Comment:
It's okay according to PHP documentation: http://php.net/manual/en/migration54.new-features.php